Hazel/src/Hazel/Window.h

```diff
@@ -6,17 +6,26 @@
 #include "Hazel/Events/Event.h"
 
 namespace Hazel {
+	
+	enum WindowMode
```

Comment:
Wouldn't we concider making this an enum class so we force the use of the enum?
Also should we define the base type (for example uint8_t) instead of letting it default to uint32_t?

`enum class WindowMode : uint8_t`